src/ring_theory/algebraic.lean

```diff
@@ -116,6 +116,11 @@ begin
   exact is_integral_trans L_alg A_alg,
 end
 
+/-- If A is an algebraic algebra over K, then A is algebraic over L when L is an extension of K -/
+lemma is_algebraic_of_larger_field (A_alg : is_algebraic K A) : is_algebraic L A :=
```

Comment:
Hmm, I'm wondering what the best name is. How about `is_algebraic.extend_base`?
Also, I think `L` should be an explicit variable. So that one can write `have := A_alg.extend_base L`.